pulumi/infra/observability_env_vars.py

```diff
@@ -57,13 +59,14 @@ def otel_config(
     config:
       scrape_configs:
         - job_name: 'nomad-server'
-          scrape_interval: 10s
```

Comment:
The scrape interval has to be greater than the scrape timeout, otherwise the scraping times out before any scraping is actually conducted.
Essentially these are absolute times, scrape_timeout is not relative to the scrape_interval